radish/documentors/__main__.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module provides a script to document Radish step implementations
+"""
+
+from sys import exit
+
+from os.path import split
+
+from docopt import docopt
+
+from radish import __VERSION__
+
+from radish.documentors.sphinx.multipage import MultiPageSphinxDocumentSet
+from radish.documentors.sphinx.singlepage import SinglePageSphinxDocument
+from radish.documentors.steps_writer import StepsDocumentWriter
+
+from radish.errororacle import error_oracle, enable_excepthook
+
+from radish.exceptions import RadishError
+
+from radish.loader import load_module_assets
+
+from radish.main import setup_config
+
+from radish.stepregistry import StepRegistry
+
+
+# ........................................................................... #
+@error_oracle
+def main():
+    usage = """
+Usage:
+    radish-docs sphinx <module-path>
+        [-o=<folder-path> | --output-folder=<folder-path>]
+        [--index-file-name=<file-path> ]
+        [-s=<file-path> | --single-file=<file-path>]
+        [-c | --print-to-console]
+        [--label-prefix=<label-prefix>]
+        [-r | --recreate]
+        [--no-ansi]
+    radish-docs (-h | --help)
+    radish-docs (-v | --version)
+
+Commands:
+    sphinx
+      create Sphinx documentation
```

Comment:
So the problem I encountered is that if I keep everything on the same line it really goes of the screen and wraps oddly. For example
```
Options:
    -o=<folder-path>, --output-folder=<folder-path> location of the output folder, by default `_build` in the current working folder
    --index-file-name=<file-name> optional, name for the index file created by `--output-folder`, by default "index.rst". This file is placed in the output folder.
```

I'll check if I was wrong and docopt handles. Stand by for follow up.